README.rst

```diff
@@ -1,4 +1,39 @@
-Here there will be documentation for the generator
+Sphinx GitHub Pages Generator
+*****************************
 
-..
-   comment #TODO
\ No newline at end of file
+.. _docu_start:
+
+Overview
+========
+
+This project provides a python library for accessing automatic generation of sphinx GitHub pages 
+for a given repository. It is intended for th use with a Continuous Integration solutions.
+Developed using GitHub Actions.
+
+In a Nutshell
+=============
+
+Prerequisites
+-------------
+
+- Python 3.8+
+
+Installation
+-------------
+#todo fix link
+Install the package from Github via `pip`:: #todo make pip package
+
+    pip install -e git://github.com/exasol/bucketfs-utils-python.git@{tag name}#egg=exasol-bucketfs-utils-python
+
+Documentation
+-------------
+
+`Documentation for the latest release <https://exasol.github.io/sphinx-github-pages-generator/main>`_ is hosted on the Github Pages of this project.
+
+Features
+========
+
+* Build html documentation files using Sphinx
+* Choose which remote branch to generate the documentation from
+* Choose between automatically committing or committing and pushing the generated files to your GitHub Pages branch.
```

Comment:
For me it's not really clear what the difference here is....

Reply:
changed

Reply:
Could you please check if it is more clear now?